Babylon/commands/api/solutions/create.py

```diff
@@ -37,35 +39,39 @@
 )
 @option("--select", "select", is_flag=True, default=True, help="Save this new solution in configuration")
```

Comment:
in other commands, select option has been removed, on every creation the created resource is saved in state and the user is informed in log :slightly_smiling_face: